src/services/functionAppService.ts

```diff
@@ -46,20 +47,27 @@ export class FunctionAppService extends BaseService {
   }
 
   public async deleteFunction(functionApp: Site, functionName: string) {
+    Guard.null(functionApp);
+    Guard.empty(functionName);
+
     this.serverless.cli.log(`-> Deleting function: ${functionName}`);
     const deleteFunctionUrl = `${this.baseUrl}${functionApp.id}/functions/${functionName}?api-version=2016-08-01`;
 
     return await this.sendApiRequest("DELETE", deleteFunctionUrl);
   }
 
   public async syncTriggers(functionApp: Site) {
+    Guard.null(functionApp);
+
     this.serverless.cli.log("Syncing function triggers");
 
     const syncTriggersUrl = `${this.baseUrl}${functionApp.id}/syncfunctiontriggers?api-version=2016-08-01`;
```

Comment:
Seeing the `api-version` everywhere makes me twitch a little bit, but probably more readable than trying to have some `buildUrl("syncfunctiontriggers") { return ...}` . And if it ain't broke...

Reply:
There are many different versions and they are not consistent enough to warrant putting into constants right now.  Happy to do this later if needed.